### ISSUE TITLE
Does this work with generated URLs?

### ISSUE THREAD

**Issue by merwok** [hellysmile/django-activeurl#52]
Hello!  Just looking at this project, I noticed that the README only shows examples of hard-coded URLs (`/menu/`), which seems to me unusual: I am used to projects with routes defined in urlconf, and all links are generated with `reverse` / `url`.  Does activeurl work if the templates use `url`?  what if there is `href="{{ some_variable}}"`?

If the answer is yes, then I suggest making it clear in the doc!
Thanks for considering this

**Comment by hellysmile** [comment]
Yep, that should work, and you are more than welcome to make PR to improve documentation
